README.md

```diff
@@ -46,7 +46,7 @@ Current Canopy-App components:
 
     - `canopy_phot_mod.F90`
 
-4.  In-Canopy vertical biogenic emissions (kg m-3 s-1). Note the units are in per m3 (layer volume) for vegetation type in grid cell/point.  Based on MEGANv2 and v3 (Guenther et al., 2012), and using both Clifton et al. (2021) and Silva et al. (2020) parameterizations.
+4.  In-Canopy vertical biogenic emissions (kg m-3 s-1). Note the units are in per m3 (layer volume) for vegetation type in grid cell/point. To get 2D total flux of biogenic emissions per m2 for each grid cell, the emissions profile would need to be integrated through the canopy layers, i.e., E*dz (or simply summed if dz is constant), and then multiplied by the fractional grid box areal coverage for each vegetation types.  For the current dominant vtype approach as input to Canopy-App, this multiplicative fraction = 1.  Based on MEGANv2 and v3 (Guenther et al., 2012), and using both Clifton et al. (2021) and Silva et al. (2020) parameterizations.
```

Comment:
$\sum_i E_i \delta z_i = \delta z \sum_i E_i$ if $\delta z_i = \delta z$ (constant), i.e. still multiplying dz, but you can do it after the sum if dz is constant.

Reply:
@zmoon I have modified the README.